rdkit/Chem/AtomPairs/Utils.py

```diff
@@ -153,7 +153,7 @@ def NumPiElectrons(atom):
         # unsaturations (valence - degree):
         res = atom.GetExplicitValence() - atom.GetNumExplicitHs()
         if res < atom.GetDegree():
-            raise ValueError("explicit valence exceeds atom degree")
+            raise ValueError("The explicit valence exceeds atom degree")
```

Comment:
The same error message is used in the C++ code (Code/GraphMol/Fingerprints/FingerprintUtil.cpp). 

Please keep messages unchanged. When working on a shared code base, it's better to adjust to its conventions